src/test-utils/dom/copy-to-clipboard/index.ts

```diff
@@ -12,9 +12,9 @@ export default class CopyToClipboardWrapper extends ComponentWrapper {
     return this.findComponent(`.${ButtonWrapper.rootSelector}`, ButtonWrapper)!;
   }
 
-  findStatusText(): null | ElementWrapper {
+  findStatusText(options = { popoverRenderWithPortal: false }): null | ElementWrapper {
     return this.findComponent(`.${PopoverWrapper.rootSelector}`, PopoverWrapper)!.findContent({
-      renderWithPortal: true,
+      renderWithPortal: options.popoverRenderWithPortal,
```

Comment:
I believe this one requires a dry run as we change the default from true to false.

Reply:
I did the dry-run (6937062628), also added the dry-run id to the PR description. I'll reach out to you via slack with more details about.